src/routers/handlers/email/changeEmailAddress.ts

```diff
@@ -91,6 +91,8 @@ export class ChangeEmailAddressHandler extends GenericHandler {
     logger.info(`POST request to serve change registered email address page`);
 
     const session: Session = req.session as Session;
+
+    this.viewData.companyEmailAddress = session.getExtraData(REGISTERED_EMAIL_ADDRESS);
```

Comment:
Looks good - can we update `changeEmailAddress.unit.ts` too to ensure that this will throw a test error if not in `viewData`?

Reply:
https://github.com/companieshouse/registered-email-address-web/blob/b8a4e051b767fca84098286f88549f2cee85679e/test/src/routers/handlers/email/changeEmailAddress.unit.ts#L155